newrelic/core/attribute.py

```diff
@@ -89,6 +89,7 @@
 
 MAX_NUM_USER_ATTRIBUTES = 128
 MAX_ATTRIBUTE_LENGTH = 255
+MAX_ML_ATTRIBUTE_LENGTH = 4096
```

Comment:
When I ran this through to the UI it showed an off-by-1 error. We just need to change this to 4095.
```suggestion
MAX_ML_ATTRIBUTE_LENGTH = 4095
```